docs/hardfork-activation-inheritance.md

```diff
@@ -9,7 +9,7 @@ If and when a frontier chain becomes a standard chain, a `superchain_time` key/v
 There is a mechanism whereby hardfork activation times are set _superchain-wide_ in the `superchain.toml` configuration file present for each superchain target, but then conditionally propagated to all standard chains in that superchain when configurations are loaded by OP Stack software. This allows a single location to be updated and to coordinate many chains' hardfork activations.
 
 For a chain to "receive" a particular default (superchain-wide) hardfork activation time, the following conditions must hold:
-* It must be a standard chain with `superchain_time` set (only standard chains have this field)
+* It must be a standard chain with `superchain_time` set (only standard chains have this field automatically, but it is OK for any chain to opt in)
```

Comment:
Maybe even something like
```suggestion
* It must have the `superchain_time` set. All superchain-wide hardfork activations will be inherited starting from this timestamp. 
```
It doesn't really have anything to do with "standard chain"s, right? May also merge this with another bullet point.

Reply:
Good catch, I hadn't spotted the earlier reference to "standard".